bom/application/pom.xml

```diff
@@ -2831,7 +2836,7 @@
             </dependency>
             <dependency>
                 <groupId>io.quarkus.http</groupId>
-                <artifactId>quarkus-http-websockets-jsr</artifactId>
+                <artifactId>quarkus-http-websocket-vertx</artifactId>
```

Comment:
Sorry for the nitpicking but it it expected one of of the artifact is `websockets` with an s and the other `websocket` singular?

Reply:
I see you removed the other extension from the tree so fine by me.